python/py-photomosaic/Portfile

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        worldveil photomosaic d24fb85
+version             2020.06.03
+revision            0
+name                py-${github.project}
+categories-append   graphics
+
+license             none
+maintainers         nomaintainer
+platforms           darwin
+
+description         Creating fun photomosaics, GIFs, and murals from \
+                    your family pictures using ML & similarity search
+
+long_description    ${description}
+
+checksums           rmd160  3c4ba6aa62d34befeac9a29349d9f70292cb284b \
+                    sha256  a535bd251ba7a0a58ffbc54267dd3d216039444c27fe699a3780962c3b74854d \
+                    size    206597225
+
+python.versions     37 38
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
+
+# Add this below after https://github.com/macports/macports-ports/pull/7733 merged
```

Comment:
please revise the part below - it looks like the PR has been merged, so I guess these dependencies should be added now?

Reply:
Thanks, done.

Reply:
Also see https://github.com/worldveil/photomosaic/issues/5